src/videoosd.html

```diff
@@ -1,27 +1,19 @@
 <div id="videoOsdPage" data-role="page" class="page libraryPage" data-backbutton="true">
-    <div class="pageContainer flex">
-
-    </div>
-    <div class="upNextContainer hide">
-    </div>
-
+    <div class="pageContainer flex"></div>
+    <div class="upNextContainer hide"></div>
     <div class="videoOsdBottom videoOsdBottom-maincontrols">
-
-        <div class="osdPoster">
-
-        </div>
+        <div class="osdPoster"></div>
         <div class="osdControls">
-
             <div class="osdTextContainer osdMainTextContainer">
                 <h3 class="osdTitle"></h3>
                 <div class="osdMediaInfo"></div>
                 <div class="osdMediaStatus hide">
-                    <i class="md-icon animate">autorenew</i> <span>${FetchingData}</span>
+                    <i class="md-icon animate">autorenew</i>
+                    <span>${FetchingData}</span>
```

Comment:
is indentation correct here?

Reply:
It is now, it definitely wasn't before though.

Reply:
Right, I didn't notice closing `</i>`